src/components/content/MessageWithLink.tsx

```diff
@@ -1,13 +1,14 @@
-import { Box } from '@mui/material';
+import { Box, SxProps } from '@mui/material';
 import ExternalLink from 'components/shared/ExternalLink';
 import { FormattedMessage, useIntl } from 'react-intl';
 
 interface Props {
     messageID: string;
     link?: string;
+    linkSx?: SxProps;
```

Comment:
We should replace this with just passing any of the `ExternalLinks` props in and spreading them into the component.